### PR TITLE
[Meeting Extensibility] Content Bubble Bot CSharp

### PR DESCRIPTION
**Pull request by Trinetra-MSFT**
## Description 

Created a Content Bubble Bot in meeting chat having an Adaptive Card sent by Bot , same Adaptive Card displayed in Content Bubble.

**Language Used:** C#

## Testing

Interaction with Content Bubble Bot in Microsoft Teams Meeting chat
image

### CODE REVIEW

File: samples/apps-in-meeting/csharp/README.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+2. Navigate to app registration page, in the overview section, copy and save the **Application (client) ID**. You’ll need those later when updating your Teams application manifest and in the appsettings.json.
+3. Under **Manage**, select **Expose an API**. 
+4. Select the **Set** link to generate the Application ID URI in the form of `api://{AppID}`. Insert your fully qualified domain name (with a forward slash "/" appended to the end) between the double forward slashes and the GUID. The entire ID should have the form of: `api://fully-qualified-domain-name/{AppID}`
+    * ex: `api://%ngrokDomain%.ngrok.io/00000000-0000-0000-0000-000000000000`.
+5. Navigate to **API Permissions**, and make sure to add the follow permissions:
+-   Select Add a permission
+-   Select Microsoft Graph -\> Delegated permissions.
+    * User.Read (enabled by default)
+    * email
+    * offline_access
```
Comment (Wajeed-msft) [inline samples/apps-in-meeting/csharp/README.md:39]: These look like steps for SSO. Could you please add relevant steps for this sample?

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]

File: samples/apps-in-meeting/csharp/Content Bubble/Bots/ContentBubbleBot.cs [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+        {
+            Activity activity = MessageFactory.Text("**Please provide your valuable feedback**");
+
+            activity.ChannelData = new TeamsChannelData
+            {
+                Notification = new NotificationInfo()
+                {
+                    AlertInMeeting = true,
+
+                    ExternalResourceUrl = "https://teams.microsoft.com/l/bubble/<<APP-ID>>?url=<<BASE-URL>>/FirstPage&height=<HEIGHT>&width=<WIDTH>&title=ContentBubble&completionBotId=<<BOT-ID>>"
```
Comment (Wajeed-msft) [inline samples/apps-in-meeting/csharp/Content Bubble/Bots/ContentBubbleBot.cs:149]: Please fill in the  AppId, Based URL from config file. Also, set fixed height and width.

File: samples/apps-in-meeting/csharp/Content Bubble/Pages/Shared/_EmbedPage.cshtml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+﻿<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (Trinetra-MSFT) [reply samples/apps-in-meeting/csharp/Content Bubble/Pages/Shared/_EmbedPage.cshtml]: Yes this is requires to load the HTML

File: samples/apps-in-meeting/csharp/Content Bubble/Bots/ContentBubbleBot.cs (hunk 2 above)
Comment (Trinetra-MSFT) [reply samples/apps-in-meeting/csharp/Content Bubble/Bots/ContentBubbleBot.cs]: Done

File: samples/apps-in-meeting/csharp/README.md (hunk 1 above)
Comment (Trinetra-MSFT) [reply samples/apps-in-meeting/csharp/README.md]: Removed this added specific steps for configuring content bubble bot

**Comment by Trinetra-MSFT** [comment]
@Wajeed-msft: Review comments are updated, please check this

**Review by Wajeed-msft — APPROVED** [review]

**Review by asnow003 — APPROVED** [review]

**Pull request merged**